### PR TITLE
Update tutorial.md

### PR DESCRIPTION
**Pull request by malevolentninja**
I have added the accessibility information to the further reading section. For some reason my links aren't converting themselves. If anyone has feedback regarding the amount of content I have added, that would be great as I might have added a bit too much?

### CODE REVIEW

File: html/lesson6/tutorial.md [hunk 1]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
 Now that you are familiar with HTML & CSS, how about you create your own portfolio site. Think about the pages that you may include, such as home, about me and a portfolio or gallery page. You may even want to include a downloadable CV.
+
+## Further Reading:
+### Accessibility
+
+Some of your users might be using screenreaders or keyboards to navigate across your page.
+How do you ensure your website is available to them? By following a few guidelines and integrating techniques.
+
+#### A few names acronyms to know:
+- (ARIA) Accessible Rich Internet Applications are set of techniques and additions to HTML to allow additional markup to be interpreted by accessibility devices
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: missing word:
> are **a** set of techniques

also, missing full stop at the end of the sentence:
> interpreted by accessibility devices.

File: html/lesson6/tutorial.md [hunk 2]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
+- (WCAG) Website Content Accessibility Guidelines contain different compliance levels (A, AA and AAA) and lay down the “rules”to make a website or application accessible.
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: Missing space between words:
> “rules” to

File: html/lesson6/tutorial.md [hunk 3]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
+- (WAI) Web Accessibility Initiative of W3 cover standard, guidelines, rules and techniques.
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: I feel that this wording is a little confusing. How about this?
> (WAI) The Web Accessibility Initiative is a part of the W3 consortium that works on accessibility standards, guidelines, rules and techniques.

File: html/lesson6/tutorial.md [hunk 4]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
+
+#### There is a growing amount of information about accessibilty, here are few articles to get you started!
+- (Introduction to Accessibility)[https://www.w3.org/standards/webdesign/accessibility]
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: These Markdown URLs need their brackets flipped: The square bracket comes first, then the parentheses. Like this:
`[Introduction to Accessibility](https://www.w3.org/standards/webdesign/accessibility)` --> [Introduction to Accessibility](https://www.w3.org/standards/webdesign/accessibility)

File: html/lesson6/tutorial.md [hunk 5]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
 
 Now that you are familiar with HTML & CSS, how about you create your own portfolio site. Think about the pages that you may include, such as home, about me and a portfolio or gallery page. You may even want to include a downloadable CV.
+
+## Further Reading:
+### Accessibility
+
+Some of your users might be using screenreaders or keyboards to navigate across your page.
+How do you ensure your website is available to them? By following a few guidelines and integrating techniques.
+
+#### A few names acronyms to know:
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: How about changing this heading to just say,
> Acronyms to know

Also, I would include the `a11y` numeronym in this section too!

File: html/lesson6/tutorial.md [hunk 6]
```diff
@@ -143,3 +143,23 @@ This ends our sixth lesson. How did you find learning HTML and CSS? If you have
...
+- (ARIA) Accessible Rich Internet Applications are set of techniques and additions to HTML to allow additional markup to be interpreted by accessibility devices
+- (WCAG) Website Content Accessibility Guidelines contain different compliance levels (A, AA and AAA) and lay down the “rules”to make a website or application accessible.
+- (WAI) Web Accessibility Initiative of W3 cover standard, guidelines, rules and techniques.
+
+#### There is a growing amount of information about accessibilty, here are few articles to get you started!
```
Comment (richardwestenra) [inline html/lesson6/tutorial.md]: How about just,
> Articles for getting started with accessibility

?

File: html/lesson6/tutorial.md (hunk 4 above)
Comment (malevolentninja) [reply html/lesson6/tutorial.md]: Thanks for your help, I will get onto fixes these right away!

File: html/lesson6/tutorial.md (hunk 5 above)
Comment (malevolentninja) [reply html/lesson6/tutorial.md]: Good Point!

File: html/lesson6/tutorial.md (hunk 1 above)
Comment (malevolentninja) [reply html/lesson6/tutorial.md]: Thanks for pointing that out, not sure how I missed it!

File: html/lesson6/tutorial.md (hunk 3 above)
Comment (malevolentninja) [reply html/lesson6/tutorial.md]: Definitely way more clearer.

**Comment by malevolentninja** [comment]
Glad I have been able to help contribute somewhat. Please feel free to mention anything else worth including!

**Pull request merged**